core/domain/exp_services_test.py

```diff
@@ -958,6 +966,11 @@ def test_update_param_changes(self):
             self.owner_id, self.EXP_ID, _get_change_list(
                 self.init_state_name, 'param_changes', self.param_changes), '')
 
+        user_exp = user_models.ExplorationUserDataModel.get_by_id(
+            '%s.%s' % (self.owner_id, self.EXP_ID))
+        self.assertEqual(user_exp.draft_change_list, None)
```

Comment:
This test can be moved to the other test suite (to, e.g. test_default_values_of_exp_user_data_model_properties).

Also, please use self.assertIsNone() instead of self.assertEqual(), if you're comparing to None.